frontend/packages/dev-console/src/components/import/jar/section/JarSection.tsx

```diff
@@ -50,9 +50,9 @@ const JarSection: React.FunctionComponent = () => {
 
   React.useEffect(() => {
     if (fileUpload) {
-      updatedJarFile(null, fileUpload);
+      updatedJarFile(null as any, fileUpload);
```

Comment:
we could just change the event in `updatedJarFile` to be any because we don't use the event paramter